source/security/index.md

```diff
@@ -113,8 +113,75 @@ header to each JSP file
 <%@ page contentType="text/html; charset=UTF-8" %>
 ```
 
+### Defining and annotating your Action parameters
+
+> Note: Since 6.4 using `struts.parameters.requireAnnotations=true`. Or by default from 7.0.
```

Comment:
Do we have a ticket to enable this by default in S7?

Reply:
WW-5353 - but I'll wait till all other bugs in 7.0 have been resolved prior to making these changes